src_docs/developer/adr/2025004.TMXFormatAndValidation.md

```diff
@@ -0,0 +1,220 @@
+# ADR: Format Compliance and Purpose of `project_save.tmx` and TMX Validation Strategy
+
+## Status
+**Proposed** (Date: 2025-07-19)
+- Related Threads:
+    - "[OmTdev] The internal TMX has DOCTYPE and DTD" (2025-07-13 to 14)
+    - "[OmTdev] Some inconsistencies in TMX test cases and format handling" (2025-07-18 to 19)
+    - "[OmTdev] Clarifying XML Processing in OmegaT and Use of XSD over DTD" (2025-07-15 to 16)
+    - Mailing List: [SourceForge OmegaT-devel July 2025](https://sourceforge.net/p/omegat/mailman/omegat-development/thread/D67B4FB4-137E-4EFB-946C-72BAAB055845%40traductaire-libre.org/)
+
+## Context
+OmegaT stores its internal translation memory in a file called `project_save.tmx`. 
+Historically, this file has included a DOCTYPE declaration for TMX 1.1 (e.g., `<!DOCTYPE tmx SYSTEM "tmx11.dtd">`).
+While this implies TMX conformance, the file is:
+- Meant for internal use,
+- Not always valid TMX 1.1 or 1.4b,
```

Comment:
Since encapsulation is not a requirement for native codes in TMX, I don’t think we can affirm that the file is not valid TMX 1.1.

Reply:
> Since encapsulation is not a requirement for native codes in TMX, I don’t think we can affirm that the file is not valid TMX 1.1.

Consider it as valid if it makes you happy, but the fact is that any other CAT tool will interpret these tags as part of the text which is not correct. So this file is valid but not correctly readable by other CAT tools.